src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -52,7 +52,7 @@ public void getAll_invalidType() {
     @Test
     public void fill_bulldozerValue() {
         List<Object> machines = new ArrayList<>();
-        int randomSize = new Random().nextInt(100);
+        int randomSize = new Random().nextInt(10);
```

Comment:
please return this value to 100 as it was originally in test
you have replaced 100 with 10